### DISCUSSION TITLE
how to make it responsive and detect whether it's a desktop or mobile screen

### DISCUSSION THREAD

**Discussion by bobwatcherx** [zauberzeug/nicegui#476] [Q&A] [answered]
is there a way to detect if this app is running desktop or mobile .and is there a bootstrap like way to setup col example

<div class="col-md-12 col-xs-4 col-lg-4">

**Comment by rodja** [comment, marked as answer]
Just use css media queries. Also Quasar and Tailwind provide convenient classes to simplify it even further.
